strongbox.go

```diff
@@ -309,6 +333,110 @@ func smudge(r io.Reader, w io.Writer, filename string) {
 	}
 }
 
+func mergeFile() {
+	// The merge driver is expected to leave the result of the merge in the file
```

Comment:
A link to the docs will probably be helpful for people reading this in the future

```suggestion
	// docs: https://git-scm.com/docs/gitattributes#_defining_a_custom_merge_driver
	// The merge driver is expected to leave the result of the merge in the file
```

Reply:
Same as above